pilot/pkg/bootstrap/istio_ca.go

```diff
@@ -151,9 +151,9 @@ func (s *Server) EnableCA() bool {
 		// If TOKEN_ISSUER is set, we ignore the lack of mounted JWT token, it means user is using
 		// an external OIDC provider to validate the tokens, and istiod lack of a JWT doesn't indicate a problem.
 		if features.JwtPolicy.Get() == jwt.JWTPolicyThirdPartyJWT && trustedIssuer.Get() == "" {
-			log.Warnf("istiod running without access to K8S tokens (jwt path %v); disable the CA functionality",
+			log.Warnf("istiod running without access to K8S tokens (jwt path %v) ",
 				s.jwtPath)
-			return false
+			return true
```

Comment:
what is this change for?

Reply:
I'm testing by running istiod locally - with k8s access but without JWT tokens. This is also a good way to test the interception/etc - i.e. run the new DNS code on a VM or pod by adding istiod as a 'sidecar'. 

Of course the final implementation will link the library and a small subset into istio-agent.

Reply:
So @costinm will you be reverting this change before merging?